Feature/inline images as attachments

SimonBerry555 opened the pull request:
Using the 0.9.1 version of the plugin works well - I want any inline images in the document uploaded as attachments to the page.

SimonBerry555 committed:
First commit for feature.

SimonBerry555 committed:
Add test for AttachmentService

SimonBerry555 committed:
Add more tests

qwazer commented on src/main/java/com/github/qwazer/markdown/confluence/core/service/markdown/F.java:
```diff
@@ -8,8 +8,8 @@
  *
- * @param <R>
- * @param <P>
+ * @param <R> blah
```
[minor] remove or replace meaningless word blah

SimonBerry555 replied:
Done

qwazer commented on src/main/java/com/github/qwazer/markdown/confluence/core/service/AttachmentService.java:
```diff
@@ -0,0 +1,33 @@
+import java.nio.file.Path;
+
+@Service
+public class AttachmentService {
```
add logging

SimonBerry555 replied:
Done

qwazer commented on src/main/java/com/github/qwazer/markdown/confluence/core/service/MainService.java:
```diff
@@ -20,30 +25,89 @@
         }
     }
 
+    // Any images in the markdown need to be published as attachments to the page.
```
this feature and how to use 
should be part of plugin doc  (README.md)

SimonBerry555 replied:
Done

qwazer reviewed (commented):
Thanks a lot for a wonderful feature. Comments has been added from my side

qwazer commented:
commit message "First commit for feature." should be rephrased to provide useful info

qwazer commented:
OR squash all commits together

SimonBerry555 committed:
Fix issues after review.

SimonBerry555 commented:
Hi - I have addressed all the comments -would you be able to re-review and if OK merge - maybe you could squash the commits so the comment reads:
 `Add feature to process markdown inline images where the target is a file on local filesystem as attachments on the confluence page.`

Thanks

SimonBerry555 commented:
Hi,

I have made the changes you suggested, could you re-review and if you are
happy merge it into the codebase. When you merge you could squash the
commits and give a better comment - see my comment on the PR.

Once merged could you do a release of 0.9.2?

Any chance I could be added as a contributor as I have a couple of other
things I would like to add?

Thanks

On Mon, 1 Jun 2020 at 14:12, qwazer <notifications@github.com> wrote:

> OR squash all commits together
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/qwazer/markdown-confluence-gradle-plugin/pull/21#issuecomment-636853053>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AO24FLK2PFSXPNEB5SWVQETRUOSMNANCNFSM4NM7KYRQ>
> .
>
